assets/locales/active.cy.toml

```diff
@@ -536,3 +536,13 @@ one = "You’ve accepted all cookies. You can <a href=\"/cookies\">change your c
 [CookiesBannerHide]
 description = "Hide"
 one = "Hide"
+
+[CookiesPreferencesSaved]
+description = "We have saved your cookie preferences."
+one = "We have saved your cookie preference."
+other = "We have saved your cookie preferences."
+
+[CookiesPreferencesSavedAmend]
+description = "Should you ever wish to amend them, visit <strong><a href=\"/cookies\">ons.gov.uk/cookies</a></strong>."
```

Comment:
Is the wording final on this? Or just placeholder until text confirmed?
I don't think we as a formal, neutral, unbiased, factual statistical body should be using words like `wish` and `luck`. The tone doesn't fit with the rest of the website. 

I think `want` is more accurate but alternatives might be `like`, `desire`(although not a huge fan of these ones).

Reply:
Did speak to Ben about the copy and I don't think he's had any confirmation; not sure if that means there has been confirmation somewhere along the line, though. Will try and find out, though

Reply:
To summarise as per discussion, there haven't been any issues raised by anyone with regards to the wording or tone of voice, so the current wording we have should be fine